build-scripts/gulp/gen-icons-json.js

```diff
@@ -116,5 +116,10 @@ gulp.task("gen-icons-json", (done) => {
     JSON.stringify({ version: package.version, parts })
   );
 
+  fs.writeFileSync(
+    path.resolve(OUTPUT_DIR, "iconList.json"),
+    JSON.stringify(meta.map((icon) => icon.name))
```

Comment:
Let's just put the icons in here that are in the current version, so we also don't have to filter them out later.

Reply:
Is it right to import `.ts` file from src (where the deprecated icon list in located) inside a build-scripts file ?

Reply:
Was thinking we could sort this list on the `tags` field in `meta.json` So icons with the tag `Home Automation` will be shown first.

```js
icon.tags.includes("Home Automation");
```